src/formGenerator.vue

```diff
@@ -307,7 +307,7 @@ div.vue-form-generator(v-if='schema != null')
 				}
 
 				let isValid = this.errors.length == 0;
-				this.$emit("validated", isValid, this.errors);
+				this.$emit("validated", isValid, this.errors, this._uid);
```

Comment:
It looks like you're relying on the internal _uid from the Vue component, this property is undocumented and meant for internal use by Vue.  If you'd like to implement something like this, then I'd recommend you introduce a stable property and initialize it with a value we can rely on.